.github/workflows/pullrequest.yaml

```diff
@@ -0,0 +1,27 @@
+name: Build on pull request to master
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+      - name: Install dependencies
+        uses: borales/actions-yarn@v2.0.0
+        with:
+          cmd: install
+      - name: Call linter
+        uses: borales/actions-yarn@v2.0.0
+        with:
+          cmd: lint
+      - name: Build code
```

Comment:
is build needed here?

Reply:
I think so, kind of test. We will add extra step for tests once we have any.